core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -172,9 +186,10 @@ export class AdminBackendApiService {
   }
 
   private async _postRequestAsync(
-      handlerUrl: string, payload?: Object, action?: string): Promise<void> {
+      handlerUrl: string, payload?: Object, action?: string):
+      Promise<PopulateStatsResult> {
     return new Promise((resolve, reject) => {
-      this.http.post<void>(
+      this.http.post<PopulateStatsResult>(
```

Comment:
_postRequestAsync is a generic function for all post requests in the admin page.
(I think this change is not ideal)

Reply:
Fixed through #13230